core/color/functional.json

```diff
@@ -45,63 +14,95 @@
       "80": "rgba(0, 0, 0, 0.8)",
       "90": "rgba(0, 0, 0, 0.9)",
       "95": "rgba(0, 0, 0, 0.95)",
-      "100": "rgba(0, 0, 0, 1)"
+      "100": "#000000"
     },
     "blue": {
-      "05": "#EDF8FF",
-      "10": "#DBF0FF",
-      "20": "#BFE3FF",
-      "30": "#96D0FF",
-      "40": "#64B4FA",
-      "50": "#3492EB",
-      "60": "#1170CF",
-      "70": "#0353A8",
-      "80": "#063A75",
-      "90": "#0A274A",
-      "95": "#091A30",
-      "100": "#07101C"
+      "05": "#edf8ff",
+      "10": "#dbf0ff",
+      "20": "#bfe3ff",
+      "30": "#96d0ff",
+      "40": "#64b4fa",
+      "50": "#3492eb",
+      "60": "#1170cf",
+      "70": "#0353a8",
+      "80": "#063a75",
+      "90": "#0a274a",
+      "95": "#091a30",
+      "100": "#07101c"
+    },
+    "gray": {
+      "05": "#f7f7f7",
+      "10": "#ededed",
+      "20": "#dedede",
+      "30": "#c9c9c9",
+      "40": "#adadad",
+      "50": "#8f8f8f",
+      "60": "#707070",
+      "70": "#545454",
+      "80": "#3b3b3b",
+      "90": "#262626",
+      "95": "#1a1a1a",
+      "100": "#0f0f0f"
     },
     "green": {
-      "05": "#EDF7F5",
-      "10": "#CEF5EB",
-      "20": "#9FEDD8",
-      "30": "#68DEBD",
-      "40": "#3CC29A",
-      "50": "#27A17A",
-      "60": "#1D805F",
-      "70": "#185E46",
+      "05": "#edf7f5",
+      "10": "#cef5eb",
+      "20": "#9fedd8",
+      "30": "#68debd",
+      "40": "#3cc29a",
+      "50": "#27a17a",
+      "60": "#1d805f",
+      "70": "#185e46",
       "80": "#134231",
-      "90": "#0E2B20",
-      "95": "#0A1C15",
-      "100": "#07120D"
+      "90": "#0e2b20",
+      "95": "#0a1c15",
+      "100": "#07120d",
+      "105": "#4e6961"
     },
     "red": {
-      "05": "#FFF5F6",
-      "10": "#FFE8EA",
-      "20": "#FFD4DA",
-      "30": "#FFB8C1",
-      "40": "#FC8B98",
-      "50": "#F55868",
-      "60": "#DB1F2E",
-      "70": "#AB0A15",
-      "80": "#780D13",
-      "90": "#4F0E10",
-      "95": "#330D0E",
-      "100": "#1F0A0A"
+      "05": "#fff5f6",
+      "10": "#ffe8ea",
+      "20": "#ffd4da",
+      "30": "#ffb8c1",
+      "40": "#fc8b98",
+      "50": "#f55868",
+      "60": "#db1f2e",
+      "70": "#ab0a15",
+      "80": "#780d13",
+      "90": "#4f0e10",
+      "95": "#330d0e",
+      "100": "#1f0a0a"
+    },
+    "transparent": "rgba(255, 255, 255, 0)",
+    "white-alpha": {
+      "00": "rgba(255, 255, 255, 0)",
+      "05": "rgba(255, 255, 255, 0.05)",
+      "07": "rgba(255, 255, 255, 0.07)",
+      "11": "rgba(255, 255, 255, 0.11)",
+      "20": "rgba(255, 255, 255, 0.2)",
+      "30": "rgba(255, 255, 255, 0.3)",
+      "40": "rgba(255, 255, 255, 0.4)",
+      "50": "rgba(255, 255, 255, 0.5)",
+      "60": "rgba(255, 255, 255, 0.6)",
+      "70": "rgba(255, 255, 255, 0.7)",
+      "80": "rgba(255, 255, 255, 0.8)",
+      "90": "rgba(255, 255, 255, 0.9)",
+      "95": "rgba(255, 255, 255, 0.95)",
+      "100": "#ffffff"
```

Comment:
Same as in black-alpha-100: The value of this is in rgba in the Figma file. Why does it appear in hex here?